packages/react-composites/src/composites/CallComposite/CallScreen.tsx

```diff
@@ -131,6 +132,18 @@ export const CallScreen = (props: CallScreenProps): JSX.Element => {
     );
   }
 
+  // Do not show screen share button when composite is optimized for mobile unless the developer
```

Comment:
This component is already fairly large. Please extract a helper private function:

```

const disableScreehShareIfCallControlVisible = (options: boolean | CallControlOptions | undefined): boolean | CallControlOptions => {
    ...
}
```